helm/hpcc/templates/thor.yaml

```diff
@@ -270,24 +271,43 @@ data:
     apiVersion: networking.k8s.io/v1
     kind: NetworkPolicy
     metadata:
-      name: {{ $thorNetworkPolicyName }}
+      name: {{ $thorManagerNetworkPolicyName }}
     spec:
       podSelector:
         matchLabels:
-          app: thor
-          job: "_HPCC_JOBNAME_"
+          app: "_HPCC_JOBNAME_"
```

Comment:
This may work, but it's confusing. I don't know why you would not call the label that indicates the job name "job" rather than renaming it to "app"

Reply:
'or' pod label matching isn't support.
To ensure that the network policy only matches this job, I either had to change/add a consistent label name into the thor manager/worker job pods, or change/add the label in the thor-eclagent pod.

Alternatively, could leave 'job' as it was, and introduce a new 'instance' label or similar in both, which in thormanager/thorworker was same as _HPCC_JOBNAME_